airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -157,8 +157,8 @@ class ClusterGenerator:
     # pylint: disable=too-many-arguments,too-many-locals
     def __init__(
         self,
-        project_id: Optional[str] = None,
-        cluster_name: Optional[str] = None,
+        project_id: str,
```

Comment:
Should we add fallback for project ID from service account?

Reply:
Added